marketplacetests/marketplace/regions/base_region.py

```diff
@@ -0,0 +1,12 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marketplacetests.marketplace.app import Marketplace
+
+
+class BaseRegion(Marketplace):
```

Comment:
I was thinking too that we should probably rename all of these to pages from regions. Good idea for a later refactor.